### PR TITLE
Log points

### PR DESCRIPTION
**Pull request by bomsy**

### CODE REVIEW

File: assets/panel/debugger.properties [hunk 1]
```diff
@@ -490,17 +490,30 @@ editor.removeBreakpoint=Remove breakpoint
 
 # LOCALIZATION NOTE (editor.editBreakpoint): Editor gutter context menu item
 # for setting a breakpoint condition on a line.
-editor.editBreakpoint=Edit breakpoint
+editor.editBreakpoint=Edit condition
```
Comment (flodolo) [inline assets/panel/debugger.properties]: Quick note: to change strings like this, you need to use a new string ID (and updated the comment accordingly).
https://developer.mozilla.org/en-US/docs/Making_String_Changes

**Comment by bomsy** [comment]
depends on https://github.com/devtools-html/debugger.html/pull/7593 landing

**Comment by jasonLaster** [comment]
Thanks @bomsy!

File: assets/panel/debugger.properties [hunk 2]
```diff
@@ -494,13 +494,30 @@ editor.editBreakpoint=Edit breakpoint
 
 # LOCALIZATION NOTE (editor.addConditionalBreakpoint): Editor gutter context
 # menu item for adding a breakpoint condition on a line.
-editor.addConditionalBreakpoint=Add conditional breakpoint
+editor.addConditionalBreakpoint=Add condition
```
Comment (flodolo) [inline assets/panel/debugger.properties]: Sorry, but this shouldn't have passed review. String changes require new string IDs.

@jasonLaster Can we fix this ASAP? This just landed in m-c :-\

File: assets/panel/debugger.properties [hunk 3]
```diff
@@ -494,13 +494,30 @@ editor.editBreakpoint=Edit breakpoint
...
 editor.addConditionalBreakpoint.accesskey=c
 
+# LOCALIZATION NOTE (editor.editBreakpoint): Editor gutter context menu item
```
Comment (flodolo) [inline assets/panel/debugger.properties]: Wrong string ID reference

File: assets/panel/debugger.properties [hunk 4]
```diff
@@ -494,13 +494,30 @@ editor.editBreakpoint=Edit breakpoint
...
+# for setting a breakpoint condition on a line.
+editor.editConditionalBreakpoint=Edit condition
+
+# LOCALIZATION NOTE (editor.addLogPoint): Editor gutter context
+# menu item for adding a log point on a line.
+editor.addLogPoint=Add log
```
Comment (flodolo) [inline assets/panel/debugger.properties]: For the future, better to use X.label and X.accesskey as a couple (no need to change at this point)

File: assets/panel/debugger.properties [hunk 5]
```diff
@@ -494,13 +494,30 @@ editor.editBreakpoint=Edit breakpoint
...
+editor.addLogPoint.accesskey=l
+
+# LOCALIZATION NOTE (editor.editLogpoint): Editor gutter context menu item
+# for setting a log point on a line.
```
Comment (flodolo) [inline assets/panel/debugger.properties]: This would benefit from a comment explaining that "Edit" is a verb, and this doesn't mean "Log of edits".

**Comment by flodolo** [comment]
One more question: initially this PR tried to change `editor.editBreakpoint`, and now it adds a new string. Is the old string still used somewhere? Strings should be removed when they become unused.

File: src/components/Editor/GutterMenu.js [hunk 6]
```diff
@@ -84,6 +92,20 @@ export function gutterMenu({
     ...(breakpoint ? gutterItems.removeBreakpoint : gutterItems.addBreakpoint)
   };
 
+  const logPoint = {
+    accesskey: L10N.getStr("editor.addLogPoint.accesskey"),
```
Comment (flodolo) [inline src/components/Editor/GutterMenu.js]: I'm lost in this piece of code: is the accesskey reused for different labels?

**Comment by bomsy** [comment]
@flodolo i've opened https://github.com/devtools-html/debugger.html/pull/7646 to address most of the issues. thanks

**Comment by rugk** [comment]
I've tested this in Nightly and IMHO it's strange that this thing is still labelled "condition"… I mean, it's no condition, it's rather a list of things to log…

image

**Comment by bomsy** [comment]
@rugk Yeah we need to add context menu items for log points in the breakpoints list.

**Pull request merged**